packages/runtime-core/__tests__/componentPublicInstance.spec.ts

```diff
@@ -274,14 +274,82 @@ describe('component: proxy', () => {
     const app = createApp(Comp)
 
     app.mount(nodeOps.createElement('div'))
+    
+    // access 'toggle' to ensure key is cached
+    let v1 = instanceProxy.toggle()
```

Comment:
nit: can be `const`, and I would move the `expect(v1).toEqual('a')` right below. Same thing for v2 and v3